g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/RegExpValidatorUtils.java

```diff
@@ -129,6 +129,13 @@ public static boolean containsHTML(String inputStr) {
 		return false;
 	}
 
+	public static boolean startLetterAndCommValidchar(String str) {
```

Comment:
Is it to validate if string start with letter and comma? 
If yes, suggest change the method name to 'startWithLetterAndComma' is more readable.

Reply:
No, only start letter, the other method is the same style